worker/workercmd/worker.go

```diff
@@ -151,10 +155,10 @@ func (cmd *WorkerCommand) Runner(args []string) (ifrit.Runner, error) {
 
 	var members grouper.Members
 
-	if !cmd.gardenIsExternal() {
+	if !cmd.gardenServerIsExternal() {
```

Comment:
Does it make sense to treat external garden server as a runtime type as well?

Reply:
Hm this is a good point and something we missed out on. I agree that consolidating it with runtime type would be neater. The only problem is we'd have to break the existing contract, which might be okay.

@xtreme-sameer-vohra have you run into many users who use an external garden server?

Reply:
@chenbh I've opened a new issue to track this https://github.com/concourse/concourse/issues/5963 and put it in https://project.concourse-ci.org/projects/MDc6UHJvamVjdDUxMTY3MTg=. As this is a small use case and this PR is facing scope creep I suggest we deal with it later.